.github/workflows/cmake.yml

```
@@ -433,177 +425,184 @@ jobs:
 
-          - name: Ubuntu Clang MSAN
+          - name: Ubuntu Clang-20 MSAN
             os: ubuntu-latest
-            compiler: clang-15
-            cxx-compiler: clang++-15
+            compiler: clang-20
+            cxx-compiler: clang++-20
             cmake-args: -GNinja -DWITH_SANITIZER=Memory
-            packages:  ninja-build clang-15 llvm-15-tools libclang-rt-15-dev
-            gcov-exec: llvm-cov-15 gcov
+            packages:  ninja-build clang-20 llvm-20-tools libclang-rt-dev
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)